contracts/eip20/EIP20.sol

```diff
@@ -34,7 +38,8 @@ contract EIP20 is EIP20Interface {
 
     function transfer(address _to, uint256 _value) public returns (bool success) {
         //Default assumes totalSupply can't be over max (2^256 - 1).
-        //If your token leaves out totalSupply and can issue more tokens as time goes on, you need to check if it doesn't wrap.
+        //If your token leaves out totalSupply and can issue more tokens as time goes on, 
+        // you need to check if it doesn't wrap.
         //Replace the if with this one instead.
         //require(balances[msg.sender] >= _value && balances[_to] + _value > balances[_to]);
```

Comment:
Since we're linting, I think this is a good time to remove these weird "uncomment me for different functionality" lines.

Reply:
of course, what was I thinking.